### PR TITLE
Include the snap install steps for convenience

### PR DESCRIPTION
**Pull request by jocave**

### CODE REVIEW

File: terraform/README.md [hunk 1]
```diff
@@ -5,16 +5,30 @@ the Testflinger server into a dev, staging, or production environment.
...
+
+```
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo usermod -a -G snap_microk8s $USER
+sudo chown -f -R $USER ~/.kube
+newgrp snap_microk8s
+sudo microk8s enable hostpath-storage dns ingress
+sudo snap alias microk8s.kubectl kubectl
+sudo snap install juju --channel=3.1/stable
+mkdir -p ~/.local/share
```
Comment (plars) [inline terraform/README.md:24]: I don't recall ever having to do a lot of these steps, especially the one for .local/share. What's that for? Not a problem, just curious.

**Review by plars — APPROVED** [review]
Just a question, but nothing blocking. +1

File: terraform/README.md (hunk 1 above)
Comment (jocave) [reply terraform/README.md:24]: That's a workaround that can be needed if you're using juju from the snap confined

Comment (jocave) [reply terraform/README.md:24]: The other steps are similarly probably only needed if running microk8s confined.

Comment (plars) [reply terraform/README.md:24]: ah, ok. Makes sense

**Pull request merged**